mephisto/client/cli_commands.py

```diff
@@ -154,8 +154,11 @@ def get_wut_arguments(args):
                 checking_args[arg]["required"] = "[b]{requiredVal}[/b]".format(
                     requiredVal=checking_args[arg]["required"]
                 )
+            if arg == "tips_location":
```

Comment:
should we also add check against the blueprint name too? trying to think in the off-chance someone else names a field in a separate blueprint the same name as this

Reply:
It would make it much more verbose, because every blueprint uses a tips_location. It also kind of goes against the idea of the docs updating by themselves as you would have to go and add a blueprint name.

Reply:
Fair point